docs/setup.md

```diff
@@ -0,0 +1,42 @@
+# Set-up Service
+
+Clone the repository
+
+## Create a Virtual environment
+
+    python3 -m venv .venv
+
+## Enter the virtual environment
+
+...either macOS using bash:
+
+    source .venv/bin/activate
+
+...or if on Windows using Command Prompt:
+
+    .venv\Scripts\activate.bat
+
+## Install dependencies
+
+requirements-dev.txt and requirements.txt are updated using [pip-tools pip-compile](https://github.com/jazzband/pip-tools)
+To update requirements please manually add the dependencies in the .in files (not the requirements.txt files)
+Then run (in the following order):
+
+    pip-compile requirements.in
+
+    pip-compile requirements-dev.in
+
+From the top-level directory enter the command to install pip and the dependencies of the project
+
+    python3 -m pip install --upgrade pip && pip install -r requirements-dev.txt
+
+
+## Create static files
+
+If you would like to manually download the static files and extract them then run
```

Comment:
This isn't really optional I guess...? Maybe clarify by simplifying this line to "Download and extract the static files by running:"

Reply:
very true :)